### PR TITLE
add redux for party-edit component

### PR DESCRIPTION
**Pull request by sheleveister**

### CODE REVIEW

File: src/store/configureStore.prod.js [hunk 1]
```diff
@@ -1,8 +1,8 @@
 import { applyMiddleware, createStore } from 'redux';
 
-import rootReducer from '../reducers';
+import rootReducer from './reducers';
 import thunk from 'redux-thunk';
 
 export default function (initialState = {}) {
-    return createStore(rootReducer, initialState, applyMiddleware(thunk));
+  return createStore(rootReducer, initialState, applyMiddleware(thunk));
```
Comment (Diroro) [inline src/store/configureStore.prod.js]: Why spaces are better than tab??? Can'd understand...

File: src/store/actions/partiesActions.js [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+
+// Create party
+
+export function createParty(data) {
+  return (dispatch) => {
+    dispatch(createRequestStarted());
+    return http.post(`/parties`, data)
+      .then(res => {
+        dispatch(getParties());
```
Comment (Diroro) [inline src/store/actions/partiesActions.js]: We don't need to get parties here, after success creation we are redirected to parties list. 
There on mount we dispatch the same action: getParties(), so we're doing a double request.
So I think we can omit dispatch(getParties()) here.
That's why I've commented such lines before, although I had to delete them completely

File: src/store/actions/partiesActions.js [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+// Delete party 
+
+export function deleteParty(id) {
+  return (dispatch) => {
+    dispatch(deleteRequestStarted());
+
+    http.delete(`/parties/${id}`)
+      .then(res => {
+          dispatch(deleteRequestFinished());
+          dispatch(getParties());
```
Comment (Diroro) [inline src/store/actions/partiesActions.js]: Here we don't need getParties request, please look at my comment below

File: src/store/actions/partiesActions.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+// Update party
+
+export function updateParty(data, partyId) {
+  return (dispatch) => {
+    dispatch(updateRequestStarted());
+    return http.put(`/parties/${partyId}`, data)
+      .then(res => {
+        dispatch(getParties());
```
Comment (Diroro) [inline src/store/actions/partiesActions.js]: Same here

File: src/store/configureStore.prod.js (hunk 1 above)
Comment (sheleveister) [reply src/store/configureStore.prod.js]: this is just 2 spaces

Comment (Diroro) [reply src/store/configureStore.prod.js]: and can be just one tab

File: src/store/actions/partiesActions.js (hunk 2 above)
Comment (sheleveister) [reply src/store/actions/partiesActions.js]: removed

File: src/store/actions/partiesActions.js (hunk 4 above)
Comment (sheleveister) [reply src/store/actions/partiesActions.js]: removed

**Pull request merged**